hotfix/v0.13.5

damian-romero opened the pull request:
- [x] Change executing permissions to python scripts inside `scripts/`
- [x] Add Python shebang to `get_org_code_numbers`
- [x] Update README.md to reflect changes and bump version

Fixes #82

Commit:
Update README.md changelog #82

 - Change all references from:
   `python scripts/etc` to:
   `./ReQUIAM/scripts/etc`
   This reflects that scripts are now executable.

Commit:
Make scripts executable #82

 - Using `chmod +x`
 - Update `get_org_code_numbers` with shebang

astrochun commented on README.md:
```diff
@@ -276,7 +279,8 @@ We use [SemVer](http://semver.org/) for versioning. For the versions available,
 
-v0.13.0 - v0.13.4:
+v0.13.0 - v0.13.5:
+ * All scripts under the `scripts/` are now executable
```
```suggestion
 * Make scripts executable #82 
```

astrochun commented on README.md:
````diff
@@ -255,7 +258,7 @@ GrouperSuperAdmins and GrouperAdmins. If a group already exists, it will
 
 ```
-(figshare_patrons) $ python scripts/add_grouper_groups --config config/figshare.ini \
+(figshare_patrons) $ ./ReQUIAM/scripts/add_grouper_groups --config config/figshare.ini \
````
```suggestion
(figshare_patrons) $ ./scripts/add_grouper_groups --config config/figshare.ini \
```

astrochun commented on README.md:
````diff
@@ -199,7 +202,7 @@ To this end, the `user_update` script should be used. It has several features:
 
 ```
-(figshare_patrons) $ python scripts/user_update --config config/figshare.ini \
+(figshare_patrons) $ ./ReQUIAM/scripts/user_update --config config/figshare.ini \
````
```suggestion
(figshare_patrons) $ ./scripts/user_update --config config/figshare.ini \
```

astrochun commented on README.md:
````diff
@@ -156,7 +159,7 @@ To execute the script and update Grouper and EDS, include the `portal`, `quota`,
 
 ```
-(figshare_patrons) $ python scripts/script_run --config config/figshare.ini \
+(figshare_patrons) $ ./ReQUIAM/scripts/script_run --config config/figshare.ini \
````
```suggestion
(figshare_patrons) $ ./scripts/script_run --config config/figshare.ini \
```

astrochun commented on README.md:
````diff
@@ -145,7 +148,7 @@ Without the `sync` flag, the above command line will perform a
 
 ```
-(figshare_patrons) $ python scripts/script_run --test_reverse \
+(figshare_patrons) $ ./ReQUIAM/scripts/script_run --test_reverse \
````
```suggestion
(figshare_patrons) $ ./scripts/script_run --test_reverse \
```

astrochun commented on README.md:
````diff
@@ -131,7 +134,7 @@ Test command-line flags (`test` and `test_reverse`) are available to test EDS
 
 ```
-(figshare_patrons) $ python scripts/script_run --test \
+(figshare_patrons) $ ./ReQUIAM/scripts/script_run --test \
````
```suggestion
(figshare_patrons) $ ./scripts/script_run --test \
```

astrochun commented on README.md:
```diff
@@ -122,7 +125,7 @@ execute the following command:
 (figshare_patrons) $ export password="insert_password"
 (figshare_patrons) $ export persist_path="/path/to/persistent/storage"
-(figshare_patrons) $ python scripts/script_run --config config/figshare.ini \
+(figshare_patrons) $ ./ReQUIAM/scripts/script_run --config config/figshare.ini \
```
```suggestion
(figshare_patrons) $ ./scripts/script_run --config config/figshare.ini \
```

astrochun reviewed (commented):
Changes look good.  A few things.

1. I've simplified to call since ReQUIAM is the default path (./ReQUIAM/scripts -> ./scripts)
2. I'm not sure what happened but the Table of Contents was changed substantially and it should not have been. See below for what it looks like.  There are some indentation issues:

![Screen Shot 2020-11-10 at 1 41 56 PM](https://user-images.githubusercontent.com/20305734/98731178-917f4d00-235a-11eb-9c15-ebe642ead49c.png)

2. Please bump version to `v0.13.5` in `README.md`, `setup.py`, and `requiam/__init__.py`

damian-romero committed:
Apply suggestions from code review

 - Change executables instructions on README.md from `./ReQUIAM/scripts/` to `./scripts'
   (7 changes).
 - This hotfix still needs work.

Co-authored-by: Chun Ly <astro.chun@gmail.com>

Commit:
Incorporate review suggestions for PR #94

 - Fix indentation problem in README.md
 - Bump version: `v0.13.4` to `v0.13.5`

Commit:
Fix TOC `README.md` again (VS Code Extension issue)

astrochun committed:
Update author section in README.md #82

astrochun reviewed (approved):
Perfect.

Rebased some of your commits
